esp/src/src-react/routes.tsx

```diff
@@ -390,6 +390,11 @@ export const routes: RoutesEx = [
                     },
                 ]
             },
+            {
+                path: "/sasha", action: (ctx, params) => import("./components/Sasha").then(_ => {
```

Comment:
Should look more like the routing changes you'd made for the Dali admin, with child routes based on which "Tab" of the UI was selected?